viktor_table_view/table_view.py

```diff
@@ -77,3 +78,18 @@ def get_html(self) -> File:
                 template, {"table_html": self.style.to_html(table_attributes='class="table table-hover"')}
             )
         return result
+
+    def update_header_style(self):
+        """Updates header style if no header style is added to the styler"""
+        has_header_style = False
+        if self.style.table_styles:
+            for table_style in self.style.table_styles:
+                if "th" in table_style.values():
+                    has_header_style = True
+                    break
+        if not has_header_style or self.style.table_styles is None:
```

Comment:
remove this if statement and add put updating on same indent level of the for statement